packages/jmix-front-generator/src/generators/react-typescript/entity-form-wizard/template-model.ts

```diff
@@ -69,16 +68,26 @@ export const deriveTemplateModel = async (
 
 const deriveSteps = (
   answers: FormWizardAnswers, projectModel: ProjectModel, attributes: EntityAttribute[]
-): {steps: Array<FormStepConfig & FormEditorTemplateModel>} => {
-  return {
-    steps: answers.steps.map((step) => {
-      const stepAttributes = attributes.filter(({name}) => step.fieldNames.includes(name));
+): {steps: Array<FormEditorTemplateModel>} => {
+  const counterAttrAmountInStep = new Array(answers.steps).fill(0)
+
+  const countAttrAmountInStep = attributes.reduce((acc, _, index) => {
```

Comment:
I suggest to extract steps separation logic to function(s) and cover it with unit test